files/build_templates/pmon.service.j2

```diff
@@ -11,6 +11,9 @@ Before=ntp-config.service
 StartLimitIntervalSec=1200
 StartLimitBurst=3
 
+[Install]
```

Comment:
@vadymhlushko-mlnx why is this change?

Reply:
Without the `[Install] WantedBy=sonic.target` the PMON service won't start at SONiC boot time

Reply:
@vadymhlushko-mlnx you mean today, without this change, PMON start is delayed ? PMON does not start?

Reply:
Yes, without this change - `PMON` start is delayed

Reply:
@vadymhlushko-mlnx Here is systemd-analyze dump for NON-Mellanox. As long as all systemd dependencies are statisfied, systemd will launch PMON service


![image](https://github.com/sonic-net/sonic-buildimage/assets/45705344/5ec50648-d313-4cdf-9f07-d98886a0c6c7)


There is only 1 second delay and that too if you check carefully, PMON started 1 second before syncd

I don't see any reason for this change needed for systemd or why systemd will delay PMON if all dependencies are satisfied.

Reply:
Okay, this `[Install]` section was here before we introduced a PR that `delays the PMON and XCVRD`.
So, okay, I'm going to remove it, if it is no longer needed

Reply:
@prgeor fixed, please approve and merge